### ISSUE TITLE
Cannot Install ClyphX, Ableton 11.3.3

### ISSUE THREAD

**Issue by Brotuulaan** [ldrolez/clyphx-live11#60]
I've tried both 2.7.5 and 2.7.4, but neither will install into Ableton. I've tried both the current installation process listed (the same here on Github and on Ableton's help page) as well as the older method described in the 2.6.2 pdf (ProgramData/.../Resources/MIDI Remote Scripts), and neither location works for either version. This includes multiple Ableton restarts with every change to ensure nothing was missed. I even dropped the 2.7.5 folder into Ableton's browser in case that would change its awareness of files and paths, but no change. I imagine prior versions won't return different results but haven't tested any versions besides these two.

My User Library is on a separate drive from the app location. Could this interfere with adding 3rd-party scripts somehow? Everything native has worked fine with it split between two drives, and I've never had to search for projects that disappeared or whatnot.

**Comment by ldrolez** [comment]
Hi! Make sure that the ClyphX directory of the zip in right below your 'Remote Scripts' just like this:
![image](https://github.com/ldrolez/clyphx-live11/assets/2277898/b210fdd5-66e9-416c-a823-190f18103f74)

**Comment by Brotuulaan** [comment]
Oh yes, I forgot to mention that I tried moving the internal folder to that level so it was in the first folder below Remote Scripts. Though I didn't remove the three upper-level files (I think one was the manual). This wouldn't break it, would it? Is there some sort of reference telling the system if the folder doesn't match X file count, then something's wrong and don't load?

On another note, that was me moving it to the Clyphx 2.7.5 folder, so is the folder name also involved somehow in authentication?

I'll try changing both of those things to see if they broke it.

**Comment by Brotuulaan** [comment]
I moved the lower folder straight to the Remote Scripts folder and it's working now. I imagine it was the folder title that was the problem. I also tested moving those three files into a subfolder titled (Extras) so they wouldn't get lost in the mix for future reference after I forget what they were, and that worked just like putting them directly into the main folder.

It might be worth changing the folder structure to separate them below rather than above the script files so they can stay intact if users don't want to make a separate folder to store them or manually put them into the main folder after download. It would also avoid the confusion when dunces like me just try to put the whole thing in assuming it should be preset in a structure to contain everything.
